packages/deployment/app/cloudformation.template.yml

```diff
@@ -21,6 +21,15 @@ Resources:
   #################################################
   #  BEGIN
   #################################################
+
+{{#if ../es.name}}
```

Comment:
Is this working? Because I think it should be `{{#if es.name}}`, as it is for the rest of the ES related conditionals in this template.

Ntpick: I think it's clearer if this config lives next to the rest of the ES config in the template. It could just be nested under the existing conditional for ES templating.

Reply:
REALLY good catch here!!!

Ok so I originally had the service linked role in the IAMs and deployed, then moved it to app where it didn't work which was good because it's supposed to be a one-time thing you set up. Since it didn't work I didn't realize that. 

I removed it from here and updated the `CHANGELOG` with how to set up a service linked role for your account one-time. If you leave it here, it breaks your deployment: https://travis-ci.org/nasa/cumulus/jobs/522129725#L2031